### DISCUSSION TITLE
Diamond Theme:   Two Check Boxes Displayed

### DISCUSSION THREAD

**Discussion by sanfordandrew** [primefaces/community#1687] [PrimeReact Templates] [answered]
Just like the other user commented on the Apollo Theme, Diamond theme also displays check boxes incorrectly after upgrading to PrimeReact 10.6.3.

Can you change the theme.css or other files as required to correct in Diamond?

Thanks,

Andrew

**Comment by melloware** [comment, marked as answer]
you need to update your theme.css with the latest SASS SCSS from here: https://github.com/primefaces/primereact-sass-theme

That will fix your Diamond theme.
